### DISCUSSION TITLE
Failed to run "pip3.9 install --disable-pip-version-check --target . werkzeug==1.0.1"

### DISCUSSION THREAD

**Discussion by JeremyAndress** [vercel/community#2174] [Bug Report]
Hi 👋, I'm trying to make a build in my local with a project in Python(Fast Api). But when I run the command it shows me the following error:
```sh
Vercel CLI 28.20.0
WARN! Due to `builds` existing in your configuration file, the Build and Development Settings defined in your Project Settings will not apply. Learn More: https://vercel.link/unused-build-settings
Installing required dependencies...
Failed to run "pip3.9 install --disable-pip-version-check --target . werkzeug==1.0.1"
Error: spawn pip3.9 ENOENT
```

Idk why this is happening because on vercel this works fine but not in my local. This is my vercel file:
```json
{
  "devCommand": "uvicorn main:app --host 0.0.0.0 --port 3000",
  "builds": [
    {
      "src": "main.py",
      "use": "@vercel/python"
    }
  ],
  "routes": [
    {
      "src": "/(.*)",
      "dest": "main.py"
    }
  ],
  "installCommand": "pip install -r requirements.txt"
}

```

**Comment by asadabbas09** [comment]
have you fixed it? I'm getting the same error

**Comment by amanmibra** [comment]
Same here

**Comment by andnig** [comment]
Edit: Ok the issue is that the command pip3.9 is required locally - which for most systems might not be available in PATH.

**Comment by vivianedias** [comment]
Hey folks, I'm still running into this error. Any ideia how to fix it? I tried downgrading vercel cli, but still no progress

**Comment by louis-md** [comment]
Same thing here. Tried adding pip3.9 as an alias to pip3 in my $PATH, but it did not resolve the problem.

**Comment by sumersao** [comment]
I had this issue and resolved it by installing python3.9 and pip3.9 locally (via homebrew). I then made a conda env, creating it with python3.9, and checked that pip3.9 was installed in the env. Commands:

`brew install python@3.9`
`conda create -n ENV_NAME python=3.9`
`pip3.9 --version` in the env works. Then the build works.

**Comment by tianheg** [comment]
Change Node.js version from v20 to v18 fix it. Get more contexts from my repo's action log https://github.com/tianheg/til/actions
